en/guide/migrating-5.md

```diff
@@ -126,6 +126,20 @@ In Express 4.7 and Express 5 onwards, the query parser option can accept `false`
 
 <h3>Improvements</h3>
 
+<h4 id="async.request.handlers">async request handlers</h4>
+
+Request handlers and middleware can now be be `async` functions, and they will behave in the same way as non-async handlers when an error occurs.
```

Comment:
Do you mean "when an error is thrown"?

Reply:
FWIW, I personally think they both read pretty similar.

Reply:
@xjamundx are you happy to resolve this without a change? My intention behind using `when an error occurs` was just to try to keep things simple. I didn't want to confuse the reader by implying they might need to throw the error from their code (as opposed to from some code they're calling). Errors may also come from Node.js (not sure if they would be considered `thrown`).

Reply:
Here's the case I was concerned wasn't explained properly:

```js
app.get('/data', (req, res) => {
    DB.getData().then(data => { 
        res.send(data.property.that.does.not.exist)); // uncaught exception probably
    });
});
```
vs

```js
app.get('/data', async (req, res) => {
    const data = await DB.getData();
    res.send(data.property.that.does.not.exist); // caught in the normal way
});
```

> Request handlers and middleware can now be be `async` functions, and they will behave in the same way as non-async handlers when an error occurs.

Would any of this additional detail be helpful?

> Request handlers and middleware can now be be `async` functions. When an error is thrown or a rejected promise is `await`ed  inside of an async request handler, those errors will be caught and passed to the default error handler as if someone called `next(err)`. No special error handling is required.

Reply:
I think that's a good improvement. I've used your text as provided in 500e0408. If you're happy, thumbs up and I'll resolve.